frappe/__init__.py

```diff
@@ -393,8 +393,7 @@ def _raise_exception():
 	if flags.print_messages and out.message:
 		print(f"Message: {strip_html_tags(out.message)}")
 
-	if title:
-		out.title = title
+	out.title = title or _("Message", context="Default title of the message dialog")
```

Comment:
is this really required? This is causing test failures (which are expected cause they are now receiving `title`. 

If you have to keep this and failing tests need to ignore/add `title` key while comparing.

Reply:
It's a way to solve the problem described. I think the other option, loading translations to the website frontend, would be worse in terms of performance, refactoring effort, data security, etc.

I'll change the tests.